CHANGELOG.md

```diff
@@ -4,6 +4,30 @@ All notable changes to this project will be documented in this file.
 Each new release typically also includes the latest modulesync defaults.
 These should not affect the functionality of the module.
 
+## [v4.0.0](https://github.com/voxpupuli/puppet-nftables/tree/v4.0.0) (2024-08-04)
+
+[Full Changelog](https://github.com/voxpupuli/puppet-nftables/compare/v3.7.1...v4.0.0)
+
+**Breaking changes:**
+
+- Drop EOL CentOS 8 support [\#245](https://github.com/voxpupuli/puppet-nftables/pull/245) ([traylenator](https://github.com/traylenator))
+
+**Implemented enhancements:**
+
+- add support for conntrack helpers [\#207](https://github.com/voxpupuli/puppet-nftables/issues/207)
+- Reload nftables service if hash at last service load does not match [\#253](https://github.com/voxpupuli/puppet-nftables/pull/253) ([canihavethisone](https://github.com/canihavethisone))
```

Comment:
This entry could be better to mention that this is not the default and this is behaviour is behind a variable.

> New parameter `purge_unmanaged_rules` to reload nftables if configuration does not match reality.

Reply:
will update it